tests/README.rst

```diff
@@ -0,0 +1,8 @@
+Python Infrastructure Tests
+===========================
+
+Python Infrastructure tests are run on Docker containers with Kitchen Salt (https://kitchen.saltstack.com), testinfra (https://github.com/philpep/testinfra/), and pytest (https://pytest.org).
+
+1. Install Bundler (`gem install bundler`)
```

Comment:
Might want to encourage people to use `gem install --user bundler` rather than installing into system? I don't know best practices for Ruby environments

Reply:
Fixed.